### PR TITLE
Updated README for Android

### PR DESCRIPTION
**Pull request by handstandsam**
Updated the android specific documentation to call out the `targetSdk=33` crash: https://github.com/dropbox/dropbox-sdk-java/issues/406

Moved `ChangeLog.md` -> `CHANGELOG.md`

### CODE REVIEW

**Review by devPalacio — APPROVED** [review]
Great improvements to our android docs, thanks

**Review by Sony32FAT — COMMENTED** [review]
Run

File: README.md [hunk 1]
```diff
@@ -264,35 +264,71 @@ To run individual tests, use the `--tests` gradle test filter:
...
+<manifest>
+    ...
+    <queries>
+        <package android:name="com.dropbox.android" />
+    </queries>
+    ...
+</manifest>
+```
 
 ## FAQ
```
Comment (Sony32FAT) [inline README.md:333]: Run

**Review by Sony32FAT — COMMENTED** [review]
Get started

**Comment by Sony32FAT** [comment]
Start

**Comment by Sony32FAT** [comment]
Action

**Pull request merged**